### ISSUE TITLE
Unify terminology for frames in `frame_to_time` and  `get_frame`.

### ISSUE THREAD

**Issue by h-mayorquin** [catalystneuro/roiextractors#202]
Right now we have `frames` on the functions for getting time:
https://github.com/catalystneuro/roiextractors/blob/master/src/roiextractors/imagingextractor.py#L69

And `frame_idxs` in the function for getting the frames:
https://github.com/catalystneuro/roiextractors/blob/b71f17db4d3dadeaa4ae02c806c1e269f03d3d9b/src/roiextractors/imagingextractor.py#L62-L68
But both of them are referring to the same. We should use either `frames`, `frame_idxs` or `frame_indices` for the two of them.  Numpy uses `indices` for their indexing routines so maybe that's a good choice:
https://numpy.org/doc/stable/reference/arrays.indexing.html#routines-indexing

But I don't think it matters that much.

**Comment by CodyCBakerPhD** [comment]
I'd vote for `frames`

**Comment by CodyCBakerPhD** [comment]
Although, specific to the usage in `frame_to_time`, how badly do we want to mimic SpikeInterface and plan to eventually just have a single `get_times` function that either uses an underlying irregular `time_vector` or otherwise returns a `np.arange` scaled by the sampling frequency and shifted by an offset?

https://github.com/SpikeInterface/spikeinterface/blob/master/spikeinterface/core/baserecording.py#L441-L452

**Comment by h-mayorquin** [comment]
Yeah, maybe going for `get_times` for consistency is better overall.

**Comment by h-mayorquin** [comment]
OK, so we are going to use samples now because frames overlaps with how we refer to images.

https://github.com/catalystneuro/roiextractors/issues/406
